orttraining/README.md

```diff
@@ -0,0 +1,84 @@
+# Introduction

```

Comment:
though it's implied by the commands, it might be good to explicitly mention that these instructions are only for Linux

Reply:
Updated to clarify build from source for Linux.